examples/app/BUILD.bazel

```diff
@@ -38,4 +39,10 @@ ts_library(
     name = "e2e",
     testonly = 1,
     srcs = ["app_e2e_test.ts"],
+    tsconfig = "//examples:tsconfig-e2e.json",
```

Comment:
`tsconfig-e2e.json` should be use the tsconfig_rule and extend from `tsconfig.json`

Reply:
should probably rename tsconfig-e2e.json to tsconfig-test.json

Reply:
done